cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicationPolicyManager.java

```diff
@@ -20,16 +20,31 @@
 import java.util.Arrays;
 import java.util.List;
 
+/**
+ * Class to manage replication lifecycle events.
+ */
 public class ReplicationPolicyManager {
 
     private final List<Replicator> replicators = new ArrayList<Replicator>();
     private final ReplicationListener replicationListener = new ReplicationListener();
 
+    /**
+     * Interface with callback methods for replication lifecycle events.
```

Comment:
I'd change it slightly to:

> Interface with methods to receive callbacks for different termination states of the replications configured by a replication policy.

Note I haven't said `ReplicationPolicyManager` because this is now used by `ReplicationService` as well as `ReplicationPolicyManager` and on Android, you don't directly use `ReplicationPolicyManager`, so it could be a little confusing to the user.